internal-import-file/import-file-stix/src/import-file-stix.py

```diff
@@ -89,7 +86,19 @@ def _update_container(self, bundle: List, entity_id: int) -> List:
                 if "x_opencti_id" in object
                 and object["x_opencti_id"] == container["id"]
             ][0]
-            container_stix["object_refs"] = [object["id"] for object in bundle]
+            if self._contains_container(bundle):
+                self.helper.log_info("Bundle contains container.")
+                container_stix["object_refs"] = []
+                for elem in bundle:
+                    if self._is_container(elem.get("type")):
+                        container_stix["object_refs"].append(elem["id"])
+                        for object_id in elem.get("object_refs"):
```

Comment:
I have an error locally when importing a report that contains also containers which don't have any object_refs : 
`import-file-stix\\src\\import-file-stix.py\", line 97, in _update_container\n    for object_id in elem.get(\"object_refs\"):\nTypeError: 'NoneType' object is not iterable"}`

Reply:
Should be fixed

Reply:
it's fixed indeed